### ISSUE TITLE
Applying layer-wise learning rate decay with Deepspeed

### ISSUE THREAD

**Issue by hsyoon11** [microsoft/Swin-Transformer#248]
Hello, I am a member of distributed learning team on Samsung. 
Our team has interests on large parameter models, and submitted some MLPerf results also.
Since our team is interested on this work, we are working on object detection task with swin-transformer.

To use SwinV2-G, we are adopting DeepSpeed as written in paper.
According to experimental settings at Appendix, layer-wise learning rate decay is used for Stage-2 supervised pre-training.
However, throughput is degraded if we use DeepSpeed and layer-wise learning rate together.
This happens because layers with different learning-rates have different parameter groups.
DeepSpeed partitions every parameter groups separately, and it occurs too much communication calls(reduce).
This issue is also mentioned at DeepSpeed repo: https://github.com/microsoft/DeepSpeed/issues/2125

How did you apply layer-wise learning rate with using DeepSpeed?
Have you ever suffered from same problem? or have any solutions for it?
